src/windows/CameraProxy.js

```diff
@@ -310,29 +319,50 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
 
         tempFolder.createFileAsync(fileName, generateUniqueCollisionOption)
             .then(function(tempCapturedFile) {
-                capturedFile = tempCapturedFile;
-                return capture.capturePhotoToStorageFileAsync(encodingProperties, capturedFile);
+                return new WinJS.Promise(function (complete) {
+                    var imgStream = new Windows.Storage.Streams.InMemoryRandomAccessStream();
```

Comment:
Some comments on why the rotation needs to be done will be good - since its not quite obvious.